arangod/Aql/GraphNode.cpp

```diff
@@ -574,6 +586,7 @@ GraphNode::GraphNode(ExecutionPlan& plan, GraphNode const& other,
       _vocbase(other._vocbase),
       _vertexOutVariable(nullptr),
       _edgeOutVariable(nullptr),
+      _optimizedOutVariables({}),
```

Comment:
For some reason my comments have not been added to my review.

I think the OutVariables have to be copied here from the other node.

This is somehow done within TraversalNode, but not for Path nodes.
I would like it to be centralized, otherwise the PathNodes are now broken (we do not detect this as no optimizer rule uses those for now)

Reply:
Thanks, done in: 5b75a50b5ff6f4bb7df5747ce9ec045394ebc7b0